app/models/temp_site.rb

```diff
@@ -11,6 +12,12 @@ def name_and_code
     "#{name} (#{code})"
   end
 
+  def self.for_recruitment_cycle_years(recruitment_cycle_years = [])
```

Comment:
i'd prefer not to set a default value for the param - if `for_recruitment_cycle_years` is called without a`recruitment_cycle_years` param, it's an error and imho we should surface it

Reply:
done